OpenSim/Simulation/AssemblySolver.h

```diff
@@ -126,6 +106,9 @@ OpenSim_DECLARE_CONCRETE_OBJECT(AssemblySolver, Solver);
         to track a desired trajectory of coordinate values. */
     virtual void track(SimTK::State &s);
 
+    /** Read access to the underlying SimTK::Assembler. */
+    const SimTK::Assembler& getAssembler() const { return *_assembler; }
```

Comment:
No need to make a change for this PR, but in the future, we should consider throwing an exception when we mask a pointer as a reference (since references should never be null).